src/accessors/IConfigurationExtend.ts

```diff
@@ -6,6 +8,12 @@ import { ISlashCommandsExtend } from './ISlashCommandsExtend';
  * of your Rocketlet. It is provided during initialization of your Rocketlet
  */
 export interface IConfigurationExtend {
+    /** Accessor for registering the assets which need to be. */
+    readonly assets: IAssetExtend;
+
+    /** Accessor for declaring the contextual bar buttons your Rocketlet provides. */
+    readonly contextualBarButtons: IContextualBarButtonsExtend;
```

Comment:
Hopefully the mobile applications will eventually allow some configuration from the Rocketlets which means the server will need to serve some pieces of them up. Assets are going to be served up by the server and if they are going to require a user be logged in before accessing the asset, then yes they need to be registered.